04-gas/gas-section-six/gas-section-six.css

```diff
@@ -1,46 +1,151 @@
+.gas-section-six-subcontainer,
+.gas-section-six-subcontainer,
+.gas-section-six-copy-container,
+.gas-section-six-title-container,
+.gas-section-six-subcopy-container,
+.gas-section-six-paragraph-container,
+.gas-section-six-title,
+.gas-section-six-title-span-number,
+.gas-section-six-title-span-title,
+.gas-section-six-subtitle,
+.gas-section-six-paragraph{
+  border: var(--temporary-border);
+}
+
+.gas-section-six-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: var(--container-padding-mobile);
+  background-color: var(--level1-background-color);
+}
+
+@media (min-width:600px){
+  .gas-section-six-container {
+    padding: var(--container-padding-desktop);
+  }
+}
+
 .gas-section-six-subcontainer {
-    display: grid;
-    grid-template-columns: 1fr;
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: var(--subcontainer-padding-mobile);
+  background-color: var(--level2-background-color);
+}
+
+@media (min-width:600px){
+  .gas-section-six-container {
+    padding: var(--subcontainer-padding-desktop);
+  }
+}
+
+.gas-section-six-copy-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-areas: 
+  "gas-section-six-title-container"
+  "gas-section-six-subcopy-container";
+  align-content: start;
+  row-gap: var(--subcontainer-row-gap-mobile);
+  padding: var(--copy-container-padding-mobile);
+  background-color: var(--level3-background-color);
+}
+
+@media (min-width:600px){
+  .gas-section-six-copy-container{
+    grid-template-columns: repeat(2,1fr);
     grid-template-areas: 
-    "gas-section-six-title-container"
-    "gas-section-six-paragraph-container";
-    justify-content: center;
-    row-gap: var(--items-gap-mobile);
-    padding: var(--container-padding-mobile);
-}
-  
-.gas-section-six-subcontainer > * {
-    border: dotted 1px black;
-}
-
-/*@media (min-width:600px){
-    .gas-section-six-subcontainer {
-      display: grid;
-      grid-template-columns: repeat(2,1fr);
-      grid-template-areas: 
-      "gas-section-six-title-container gas-section-six-paragraph-container";
-    }
-}*/
-  
+    "gas-section-six-title-container gas-section-six-subcopy-container";
+    column-gap: var(--subcontainer-column-gap-desktop);
+  }
+}
+
+.gas-section-six-title-container {
+ grid-area: gas-section-six-title-container;
+ display: grid;
+ align-content: start;
+}
+
+@media (min-width:600px){
 .gas-section-six-title-container {
-    grid-area: gas-section-six-title-container;
-    display: grid;
-    justify-content: center;
-    align-content: center;
-    text-align: center;
-    font-size: var(--title-size-mobile);
+  align-content: center;
+ }
 }
 
 .gas-section-six-title {
-    background-color: var(--title-background-color-mobile);
+  display: flex;
+  flex-direction: column;
+  row-gap: var(--title-container-row-gap-mobile);
```

Comment:
thinking about this...
i am not sure about the way we are calling this..
because the name suggests it applies to the title-container element
but really it applies to the title element

so maybe make a gas specific one(?) i dont know.
or make a title one... 

--title-row-gap-mobile
or
--gas-title-row-gap-mobile

Reply:
i will change this point in all of them to gas-title row-gap-mobile